mountainsort/src/mda/diskwritemda.cpp

```diff
@@ -79,6 +79,7 @@ bool DiskWriteMda::open(int data_type, const QString &path, long N1, long N2, lo
 		mda_write_float32(zeros,&d->m_header,num_to_write,d->m_file);
 		i+=buf_size;
 	}
+	free(zeros);
```

Comment:
Traveling until Monday
On Apr 16, 2016 5:07 PM, "Sebastian Baginski" <notifications@github.com>
wrote:

> In mountainsort/src/mda/diskwritemda.cpp
> <https://github.com/magland/mountainlab/pull/33#discussion_r59974004>:
>
> > @@ -79,6 +79,7 @@ bool DiskWriteMda::open(int data_type, const QString &path, long N1, long N2, lo
> >  		mda_write_float32(zeros,&d->m_header,num_to_write,d->m_file);
> >  		i+=buf_size;
> >  	}
> > +	free(zeros);
>
> I don't really understand this code, but it leaked memory.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/magland/mountainlab/pull/33/files/a8c92bb9a1e9a369f7752ae7a0d76ccbbff1d342#r59974004>
>